tests/rptest/tests/datalake/schema_evolution_test.py

```diff
@@ -7,10 +7,17 @@
 # the Business Source License, use of this software will be governed
 # by the Apache License, Version 2.0
 import tempfile
+from time import time
+from typing import NamedTuple
+from collections.abc import Callable
+import pyhive
 
+from confluent_kafka import avro
+from confluent_kafka.avro import AvroProducer
```

Comment:
Are we planning on having protobuf support here too?

Reply:
yeah should do. the code we're testing is downstream of schema translation, but there's no harm in more verification. these tests aren't very expensive to run.

may kick it to a follow-up to unblock #24955 though

Reply:
Totally 👍